lib/services/users.pm

```diff
@@ -161,7 +161,14 @@ sub full_users_check {
         change_pwd;
         add_user;
         #verify changed password work well in the following scenario:
-        lock_screen;
+        #workaround the lock screen will cause vnc lost connection issue on
+        #SLE15+ on s390x for bsc#1182958.
+        if ((check_var('ARCH', 's390x')) && is_sle('15+')) {
+            record_soft_failure("bsc#1182958, openQA test fails in install_service - gdm crashed after lock screen on s390x");
+        }
+        else {
+            lock_screen;
+        }
```

Comment:
Just some coding style comments:
1. There is a "blank space" after `#" in some lines, but some lines there are not.
2. The first letter is capitalized in some lines, others are not.
Please keep the style insistent.
Other changes are OK.

Reply:
Thanks. While the different style is for historical reason. Anyway, I updated the comments to make them uniform.